docs/reference/job-status.rst

```diff
@@ -0,0 +1,84 @@
+.. _job_status:
+
+Checkbox Job Status
+===================
+
+A Checkbox job always receives a finishing status either automatically or
+manually. The following is a description of what each status means and when/how
+it is received.
+
+Passing Jobs
+------------
+
+The Passing outcome is the marker for a successful run. It can be assigned in
+the following situations:
+
+- Automated job returned a 0 return code
+- Automated job is marked as ``noreturn`` and the session was interrupted and
+  brought back
+- Manual job marked as passed by the user
+- Session was interrupted and when manually brought back, explicitly marking
+  the job as pass
+
+Every passing job is either marked by the following symbol ``☑`` or the text
+``job passed``. Checkbox internally uses the ``IJobResult.OUTCOME_PASS`` object
+to mark these jobs.
+
+Skipped Jobs
+------------
+
+The Skipped outcome is the marker for a job that was intentionally not started
+either by the user or Checkbox itself. This can be due to the following
+reasons:
+
+- Job with a ``require`` constraint that can not be satisfied
+- Job with a dependency on a job that is skipped itself
+- Job explicitly skipped by the user via the ``launcher``
+- Job explicitly skipped by the user via the Ctrl+C menu
+- Job explicitly skipped by the user via the resume screen
+
+Every skipped job is either marked by the following symbol ``"  "`` (white
+space) or the text ``job skipped``. Checkbox internally uses the
+``IJobResult.OUTCOME_SKIP``.
+
+Failing Jobs
+------------
+
+The Failing outcome is the marker for a failing job run. It can be assigned in
+the following situations:
+
+- Automated job returned a non-0 return code
+- Automated job is marked as ``noreturn`` and the session was interrupted and
```

Comment:
The following 3 bullets are the same as in passing jobs. looks like they should be changed to the opposite?

Reply:
sloppy copy paste, fixed